### ISSUE TITLE
 Up arrow icon size issue

### ISSUE THREAD

**Issue by diana7127** [M2Team/NanaZip#406]
In the new version 3.0, the up arrow appears too small on my screen. 

My screen details are as follows:

![PixPin_2024-05-23_22-01-38](https://github.com/M2Team/NanaZip/assets/125502871/2ab5278c-8275-44e5-986e-592c0b60bc6f)
![PixPin_2024-05-23_22-03-42](https://github.com/M2Team/NanaZip/assets/125502871/6656745b-c3a8-43f7-b25d-63879a3a9db0)

**Comment by dinhngtu** [comment]
I agree. The toolbar buttons also have no label and are difficult to use. I tried integrating the Up button into the toolbar (like File Explorer) and it looks friendlier:
<img width="691" alt="Untitled1" src="https://github.com/M2Team/NanaZip/assets/1257909/cd9b420d-ac2f-45dd-868f-234c72b285bf">

I think we can use a File Explorer-like two-toolbar layout, with navigation on top and toolbar buttons just under.

**Comment by MouriNaruto** [comment]
@dinhngtu 

But there are some issues for having a label.

For lower logical screen resolutions are not friendly because it's too long. (The typical scenario is 15.6' 1080p 125% DPI Scaling)

Kenji Mouri

**Comment by MouriNaruto** [comment]
@dinhngtu 

> I think we can use a File Explorer-like two-toolbar layout, with navigation on top and toolbar buttons just under.

Have you forgot about 7-Zip/NanaZip two-panel mode existed?

Kenji Mouri

**Comment by dinhngtu** [comment]
True, I forgot about that.

Could be interesting to add a "split panel button" (like Edge).

**Comment by FrzMtrsprt** [comment]
Update:
In the latest Nanazip 3.5 Preview 0 (3.5.1000.0), the up icon is no longer transparent

![image](https://github.com/M2Team/NanaZip/assets/15852717/d4862262-b55e-4a6b-a65b-4e69dd31f766)

**Comment by MouriNaruto** [comment]
@FrzMtrsprt It's a known issue. But this will be solved after replacing that with XAML Islands in future releases.

Kenji Mouri

**Comment by dongle-the-gadget** [comment]
Next release uses XAML Islands for the address bar so this is fixed now
<img width="557" height="230" alt="New XAML address bar with bigger up button" src="https://github.com/user-attachments/assets/ce601df2-3215-446d-adc6-7f1b084f27a0" />

**Comment by AndromedaMelody** [comment]
#642
